admin/tool/urlpreview/templates/form.mustache

```diff
@@ -0,0 +1,5 @@
+<form action="{{action}}" method="get" class="{{formClass}}">
+    <label for="url">{{urlLabel}}</label>
+    <input type="text" id="url" name="url" required>
```

Comment:
does the url get kept in the input after submission so you can submit a second time?